src/Microsoft.Diagnostics.Monitoring.WebApi/Exceptions/ExceptionsOptions.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Monitoring.Options;
+using Microsoft.Diagnostics.Monitoring.WebApi.Models;
 using System.ComponentModel;
 using System.ComponentModel.DataAnnotations;
 
-namespace Microsoft.Diagnostics.Monitoring.Options
+namespace Microsoft.Diagnostics.Monitoring.WebApi.Exceptions
```

Comment:
Why are all of these classes being moved to the WebApi assembly? They are first and foremost options. Leaving them in the Options assembly should make them available to all other assemblies.